interface/package.json

```diff
@@ -14,7 +14,9 @@
   "engines": {
     "node": ">=4.0"
   },
-  "dependencies": {},
+  "dependencies": {
+    "hclusterjs": "^1.3.0"
```

Comment:
It is possible to specify a git repository in **bower.json** for installation. I am not sure whether we should do it to have hclusterjs package in `vendors/` directory instead of in `node_modules/`.